packages/mdc-theme/_mixins.scss

```diff
@@ -18,22 +18,38 @@
 
 // Applies the correct theme color style to the specified property.
 // $property is typically color or background-color, but can be any CSS property that accepts color values.
-// $style should be one of the map keys in $mdc-theme-property-values (_variables.scss).
+// $style should be one of the map keys in $mdc-theme-property-values (_variables.scss), or a literal color value.
```

Comment:
I think @kfranqueiro briefly touched base on this, is the plan to change this into custom map?